modules/client/shared/src/main/scala/com/azavea/stac4s/api/client/SttpStacClientF.scala

```diff
@@ -40,7 +40,13 @@ case class SttpStacClientF[F[_]: MonadThrow, S: Lens[*, Option[PaginationToken]]
     Stream
       .unfoldLoopEval((baseUri.addPath("search"), initialBody)) { case (link, request) =>
         client
-          .send(basicRequest.body(request.noSpaces).post(link).response(asJson[Json]))
+          .send(
+            basicRequest
+              .post(link)
+              .contentType(MediaType.ApplicationJson)
```

Comment:
Is there a default content type that gets set, or is it just missing? I'm curious why Franklin is fine without this set but other servers aren't. Is it because we told tapir to expect json?

Reply:
@jisantuc I think the default is set to `application/octet-stream`, yea I noticed that other servers are pretty angry if it is not explicitly set to JSON. (again: https://eod-catalog-svc-prod.astraea.earth/)